### ISSUE TITLE
Blackhole community documentation

### ISSUE THREAD

**Issue by aligot-cblue** [peeringdb/peeringdb#234]
Hello all,

I think it would be great if we can add the possibility to document whether or not a network can handle BGP blackhole community. Maybe something like:

- Accept Blackhole community on peering [ yes | private peering | route server |  on request | no ]
- BGP blackhole community ID (default 666 RFC 7999 ) [int]

Best regards,

A.

**Comment by pierky** [comment]
TL/DR: +1

From the perspective of a potential consumer of this information, I also agree that it could be useful to document if a network supports handling of blackhole filtering communities or not.

Automation tools could fetch these fields from PDB and use them to decide whether or not target prefixes should be announced to that network and which community should be used to tag them.

For what regards the options that could be used to set these fields, what about something like this?

```
Accept blackhole community:

 ( ) yes, always, in any case
 ( ) on the following cases only:
    [ ] public bilateral peering
    [ ] public multilateral peering (route server)
    [ ] private peering
    [ ] customers
    [ ] ...
 ( ) on request, case by case
 ( ) no, never

BGP blackhole community:

 [ ] BLACKHOLE (RFC 7999)
 [ ] custom <string "x:y" / "x:y:z">
```

(`( )` = mutually exclusive options, `[ ]` = multiple options allowed)

It's an adjustment to the values proposed by @aligot-cblue: to make things even more complicated^Wdetailed, the first field could be added to `netixlan` records: when unset at that level, the value from the `net` record could be inherited. This would allow to have a general policy and to handle exceptions to that policy on a IX basis.

Ok, ... `set overcomplicating-things mode off` :-)

**Comment by arnoldnipper** [comment]
Same as with BFD ... has nothing to do with PeeringDB and will ''overcomplicating things''
